### ISSUE TITLE
Vote for FP16 PR

### ISSUE THREAD

**Issue by John1231983** [cavalleria/cavaface.pytorch#68]
Hi all, I am using pytorch 1.16 and it supported FP16. Could your accept the PR ? https://github.com/cavalleria/cavaface.pytorch/pull/32/files
@xsacha please update head file. Thanks

**Comment by cavalleria** [comment]
It support fp16.

**Comment by xsacha** [comment]
Yes, I saw that fp16 is in there now and forgot to close my PR. Thanks for heads up.

**Comment by John1231983** [comment]
Are you using Apex instead of AMP in pytorch? The method of @xsacha  allows us to use AMP, but we should modify something in metric.py as mention. 
@xsacha could you update the PR, it is useful for many people, included me
